### PR TITLE
hugo: utilise Homebrew's new go cache

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Ref: https://github.com/Homebrew/brew/pull/4774.

### CODE REVIEW

**Comment by commitay** [comment]
`mkcert` supports modules if you want to add it to this PR?

**Comment by DomT4** [comment]
Will look into it, thanks!

Need to fix something up on the `brew` side first; discovered a bug 😆.

File: Formula/hugo.rb [hunk 1]
```diff
@@ -16,7 +16,9 @@ class Hugo < Formula
   depends_on "go" => :build
 
   def install
+    ENV["GOPATH"] = "#{HOMEBREW_CACHE}/go_cache"
```
Comment (commitay) [inline Formula/hugo.rb]: Do we want to follow the style used by `glide` formulae?

```
ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
```

**Comment by DomT4** [comment]
Blocked on https://github.com/Homebrew/brew/pull/4778.

File: Formula/hugo.rb (hunk 1 above)
Comment (DomT4) [reply Formula/hugo.rb]: Aye, that's probably a good step 👍.

**Pull request closed without merging**